src/app/modules/loose-components.module.ts

```diff
@@ -0,0 +1,483 @@
+    OrganizationPlansComponent,
+    OrganizationsComponent,
+    OrganizationSubscriptionComponent,
+    OrgAttachmentsComponent,
+    OrgBulkConfirmComponent,
+    OrgBulkRemoveComponent,
+    OrgBulkStatusComponent,
+    OrgCiphersComponent,
+    OrgCollectionAddEditComponent,
+    OrgCollectionsComponent,
+    OrgEntityEventsComponent,
+    OrgEntityUsersComponent,
+    OrgEventsComponent,
+    OrgExportComponent,
+    OrgExposedPasswordsReportComponent,
+    OrgGroupAddEditComponent,
+    OrgGroupsComponent,
+    OrgImportComponent,
+    OrgInactiveTwoFactorReportComponent,
+    OrgManageCollectionsComponent,
+    OrgManageComponent,
+    OrgPeopleComponent,
+    OrgPoliciesComponent,
+    OrgPolicyEditComponent,
+    OrgResetPasswordComponent,
+    OrgReusedPasswordsReportComponent,
+    OrgSettingComponent,
+    OrgToolsComponent,
+    OrgTwoFactorSetupComponent,
+    OrgUnsecuredWebsitesReportComponent,
+    OrgUserAddEditComponent,
+    OrgUserConfirmComponent,
+    OrgUserGroupsComponent,
+    OrgWeakPasswordsReportComponent,
+    GeneratorComponent,
+    PasswordGeneratorHistoryComponent,
+    PasswordGeneratorPolicyComponent,
+    PasswordRepromptComponent,
+    PasswordStrengthComponent,
+    PaymentComponent,
+    PersonalOwnershipPolicyComponent,
+    PreferencesComponent,
+    PremiumBadgeComponent,
+    PremiumComponent,
+    ProfileComponent,
+    ProvidersComponent,
+    PurgeVaultComponent,
+    RecoverDeleteComponent,
+    RecoverTwoFactorComponent,
+    RegisterComponent,
+    RemovePasswordComponent,
+    ReportCardComponent,
+    ReportListComponent,
+    ReportsComponent,
+    RequireSsoPolicyComponent,
+    ResetPasswordPolicyComponent,
+    ReusedPasswordsReportComponent,
+    SecurityComponent,
+    SecurityKeysComponent,
+    SendAddEditComponent,
+    SendComponent,
+    SendEffluxDatesComponent,
+    SendOptionsPolicyComponent,
+    SetPasswordComponent,
+    SettingsComponent,
+    ShareComponent,
+    SingleOrgPolicyComponent,
+    SponsoredFamiliesComponent,
+    SponsoringOrgRowComponent,
+    SsoComponent,
+    TaxInfoComponent,
+    ToolsComponent,
+    TwoFactorAuthenticationPolicyComponent,
+    TwoFactorAuthenticatorComponent,
+    TwoFactorComponent,
+    TwoFactorDuoComponent,
+    TwoFactorEmailComponent,
+    TwoFactorOptionsComponent,
+    TwoFactorRecoveryComponent,
+    TwoFactorSetupComponent,
+    TwoFactorVerifyComponent,
+    TwoFactorWebAuthnComponent,
+    TwoFactorYubiKeyComponent,
+    UnsecuredWebsitesReportComponent,
+    UpdateKeyComponent,
+    UpdateLicenseComponent,
+    UpdatePasswordComponent,
+    UpdateTempPasswordComponent,
+    UserBillingComponent,
+    UserLayoutComponent,
+    UserSubscriptionComponent,
+    VaultTimeoutInputComponent,
+    VerifyEmailComponent,
+    VerifyEmailTokenComponent,
+    VerifyMasterPasswordComponent,
+    VerifyRecoverDeleteComponent,
+    WeakPasswordsReportComponent,
+  ],
+})
+export class LooseComponentsModule {}
```

Comment:
I'm not a big fan of this module name but I'm struggling to think of an alternative.
`MiscellaneousComponentsModule` feels a _little_ better to me

Reply:
I want the name to give the impression that this is a deprecated way of doing things, and in my opinion putting "Misc" on classes usually encourages people to use them more than it discourages.

Reply:
That's a good point!